docs/snapshotting/network-for-clones.md

```diff
@@ -142,6 +142,65 @@ Otherwise, packets originating from the guest might be using old Link Layer
 Address for up to arp cache timeout seconds. After said timeout period,
 connectivity will work both ways even without an explicit flush.
 
+### Renaming host device names
+
+In some environments where the jailer is not being used, restoring a snapshot
+may be tricky because the tap device on the host will not be the same as the tap
+device that the original VM was mapped to when it was snapshotted, as when the
+tap device come from a pool of such devices.
```

Comment:
I don't quite understand the last part of this sentence. Should it be

```suggestion
In some environments where the jailer is not being used, restoring a snapshot
may be tricky because the tap device on the host will not be the same as the tap
device that the original VM was mapped to when it was snapshotted, for example when the
tap device comes from a pool of such devices.
```

?

Reply:
I think so. I've amended it with your suggested changes.